src/it/java/io/deephaven/benchmark/tests/standard/sort/SelectDistinctTest.java

```diff
@@ -1,5 +1,5 @@
 /* Copyright (c) 2022-2023 Deephaven Data Labs and Patent Pending */
-package io.deephaven.benchmark.tests.standard.select;
+package io.deephaven.benchmark.tests.standard.sort;
```

Comment:
Why sort? Select distinct is actually an aggregation.

Reply:
Moved